### ISSUE TITLE
Reset progress causes crash

### ISSUE THREAD

**Issue by timrae** [ankidroid/Anki-Android#4571]
This is with Android 7.1

```bash
java.lang.IllegalStateException: Fragment null must be a public static class to be properly recreated from instance state.
at android.support.v4.app.BackStackRecord.doAddOp(BackStackRecord.java:384)
at android.support.v4.app.BackStackRecord.add(BackStackRecord.java:363)
at android.support.v4.app.DialogFragment.show(DialogFragment.java:158)
at com.ichi2.anki.AnkiActivity.showDialogFragment(AnkiActivity.java:348)
at com.ichi2.anki.NoteEditor.onOptionsItemSelected(NoteEditor.java:864)
at android.app.Activity.onMenuItemSelected(Activity.java:3204)
at android.support.v4.app.FragmentActivity.onMenuItemSelected(FragmentActivity.java:408)
at android.support.v7.app.AppCompatActivity.onMenuItemSelected(AppCompatActivity.java:198)
```

**Comment by timrae** [comment]
Please check the latest beta version for a fix:
https://ankidroid.org/docs/manual.html#betaTesting

**Comment by snowtimeglass** [comment]
Thank you. The bug has been fixed on my device (Android 4.3)
